src/main/java/neqsim/PVTsimulation/simulation/SaturationTemperature.java

```diff
@@ -31,7 +31,11 @@ public SaturationTemperature(SystemInterface tempSystem) {
    * @return a double
    */
   public double calcSaturationTemperature() {
-    getThermoSystem().isImplementedCompositionDeriativesofFugacity(false);
+    boolean changed = false;;
```

Comment:
```suggestion
    boolean isMultiPhaseCheckChanged = false;
```
Just a suggestion. Was not immediately clear to me, what "changed" at first